cmd/cloudinitexecute/cloudinitexecute.go

```diff
@@ -84,6 +85,24 @@ func ApplyConsole(cfg *rancherConfig.CloudConfig) {
 			continue
 		}
 
+		targetDir := mount[1]
+		if mount[3] == "bind" {
+			fileInfo, err := os.Stat(mount[0])
+			if err != nil {
+				log.Errorf("Unable to find %s: %v", mount[0], err)
+				continue
+			}
+			if mode := fileInfo.Mode(); !mode.IsDir() {
+				targetDir = filepath.Dir(mount[1])
+			}
+		}
+		if targetDir != "" {
```

Comment:
Excellent question; but, util.Mount() is just a wrapper around mount(2). We'll still need to make sure the target path exists, or the parent of the target path in the case of a bind mount. That makes this a little more complex than I'd prefer.

Converting from exec.Command() to util.Mount() seems like it should be a separate change that would require careful investigation.